yt/yt/server/query_tracker/yql_engine.cpp

```diff
@@ -88,6 +88,7 @@ class TYqlQueryHandler
         , Settings_(ConvertTo<TYqlSettingsPtr>(SettingsNode_))
         , Stage_(Settings_->Stage.value_or(Config_->Stage))
         , ExecuteMode_(Settings_->ExecuteMode)
+        , Secrets_(ConvertTo<std::optional<std::vector<TQuerySecretPtr>>>(activeQuery.Secrets).value_or(std::vector<TQuerySecretPtr>()))
```

Comment:
Let's maybe create a static function for this conversion? This line is quite long.